src/nullablevector.jl

```diff
@@ -194,8 +185,28 @@ collection to `append!`.
 function Base.append!(X::NullableVector, items::AbstractVector)
     old_length = length(X)
     nitems = length(items)
+    items_copy = convert(typeof(X), items)
     resize!(X, old_length + nitems)
-    X[old_length + 1:end] = items[1:nitems]
+    X[old_length+1:end] = items_copy
+    return X
+end
+
+@doc """
```

Comment:
We should really remove all docstrings which are redundant with Julia Base (and make PRs to improve them were needed). But for now probably better follow the current convention.